packages/coding-components/templates/react/package.json

```diff
@@ -8,20 +8,18 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@arcgis/coding-components-react": ">=4.28.0-beta.39 < 4.29",
-    "@arcgis/core": "~4.28.0",
-    "@esri/calcite-components-react": "^1.9.2",
-    "react": "18.2.0",
-    "react-dom": "18.2.0"
+    "@arcgis/coding-components-react": ">=4.29.0-beta.100 < 4.30",
+    "@arcgis/core": ">=4.29.0-next.20240207",
```

Comment:
Consider just using `@next` for versioning `@arcgis/core` for the various samples.